### PR TITLE
Fix project travis and tox builds

### PR DESCRIPTION
**Pull request by AlmogCohen**
Following the accepted PRs, I propose this PR that basically fix the build issues with this project by fixing the tox.ini file to properly work with `pipenv`.

### CODE REVIEW

**Comment by AlmogCohen** [comment]
@elgertam As you can see on my personal Travis build https://travis-ci.com/AlmogCohen/cookiecutter-pipenv/builds/98225790

This PR would fix the build for this project by using changes I've already proposed in other PRs. What I did here is to fix the pytest command run

**Comment by AlmogCohen** [comment]
Wait a moment, it is still broken for some reason. Maybe the Pipfile.lock

**Comment by AlmogCohen** [comment]
Travis build passes for this branch...which is nice to prove a point https://travis-ci.com/AlmogCohen/cookiecutter-pipenv. Ready for a merge

File: Pipfile [hunk 1]
```diff
@@ -7,16 +7,13 @@ name = "pypi"
...
-cookiecutter = ">=1.4.0"
-pytest-cookies = "==0.2.0"
-watchdog = "==0.8.3"
-alabaster = "==0.7.8"
-PyYAML = "==3.11"
-requests = ">=2.20.0"
-urllib3 = ">=1.23"
+pytest = "*"
+tox = "*"
+cookiecutter = "*"
```
Comment (AlmogCohen) [inline Pipfile:12]: @elgertam Would you agree about this removal of specific versions from the `dev-packages` of the repo? I think that the committed updated `Pipfile.lock` would be enough to let developers in the future get a working environment without forcing them to use old versions for development

Comment (AlmogCohen) [reply Pipfile:12]: And on top of that, many of the frozen versions here are *really* old and that's another way to see the disadvantage of this approach. The frozen `pytest` version here is from May 2016!!!!

Comment (AlmogCohen) [reply Pipfile:12]: and the `tox` version is from 2015... you get the point...

Comment (elgertam) [reply Pipfile:12]: I agree in principle. I'm running things locally (need to stop procrastinating on setting TravisCI for the whole repo) to verify that they're working.

I should have an answer in a few minutes.

Thank you again for keeping an eye on this.

Comment (AlmogCohen) [reply Pipfile:12]: I've added a travis build to the README to help you with the travis thingy :) Now it is shown us grey "unknown" until you'll press the button and enable the travis build on travis https://travis-ci.com/ (which I provided here as well! :P)

File: README.rst [hunk 2]
```diff
@@ -2,6 +2,12 @@
 Cookiecutter Python Package Template with Pipenv
 ================================================
 
+|travis|
+
+.. |travis| image:: https://travis-ci.com/elgertam/cookiecutter-pipenv.svg?branch=master
```
Comment (AlmogCohen) [inline README.rst:7]: Now we just need the owner of the repo to enable builds on `travis` ;) next step we could fix the PyPi deployment

Comment (AlmogCohen) [reply README.rst:7]: no pressure though

**Comment by AlmogCohen** [comment]
Pypy3.5 break the travis build at the moment as pipenv does not officially support `pypy` (which is odd). It is all mentioned here: https://github.com/pypa/pipenv/issues/3313

For now, I'll simply drop the `pypy3.5` from the build. (Another option is to pin the pipenv version but I'd rather not do that at the moment)

**Comment by AlmogCohen** [comment]
@elgertam bump... I'll just point out that the travis build of this exact branch is passing successfully on my fork https://travis-ci.com/AlmogCohen/cookiecutter-pipenv so not sure if local testing is necessary. I do respect your decision on that :)

File: Pipfile [hunk 3]
```diff
@@ -7,16 +7,13 @@ name = "pypi"
...
+pytest-cookies = "*"
+alabaster = "*"
+PyYAML = "*"
+requests = "*"
```
Comment (elgertam) [inline Pipfile]: @AlmogCohen: My apologies for taking so long.

I'd ask that the version specifier for `requests` be set back to `">=2.20.0"`. Additionally, can you pin the version of `pyyaml` to `">=4.2b1"`? These are to resolve automated GitHub security checks.

Once these changes are made and pushed, I can merge the PR.

Comment (AlmogCohen) [reply Pipfile]: I get it, will do. I think it is still a bit overkill as these are only development tools that are expected to install the latest versions upon installation

**Comment by AlmogCohen** [comment]
done

**Pull request merged**